src/platform/selfsign_capi.c

```diff
@@ -1032,7 +1032,8 @@ _IRQL_requires_max_(PASSIVE_LEVEL)
 const QUIC_CREDENTIAL_CONFIG*
 CxPlatGetSelfSignedCert(
     _In_ CXPLAT_SELF_SIGN_CERT_TYPE Type,
-    _In_ BOOLEAN IsClient
+    _In_ BOOLEAN IsClient,
+    _In_ uint8_t AllowedCiphers
```

Comment:
Instead of passing this in, let's just update the function to return a non-const, and update the caller to add the flag as necessary. This model of passing things down isn't scalable.